src/core/server/saved_objects/service/lib/repository.test.js

```diff
@@ -652,6 +652,29 @@ describe('SavedObjectsRepository', () => {
         });
       };
 
+      const bulkCreateMgetError = async (objects, options) => {
+        const multiNamespaceObjects = objects.filter(
+          ({ type, id }) => registry.isMultiNamespace(type) && id
+        );
+        if (multiNamespaceObjects?.length) {
+          const response = getMockMgetResponse(multiNamespaceObjects, options?.namespace);
+          client.mget.mockResolvedValue(
+            elasticsearchClientMock.createSuccessTransportRequestPromise(
+              { ...response },
+              { statusCode: 404 },
+              {}
```

Comment:
nit: If we aren't including the ES product header here, might be worth renaming this function for clarity so that it is obvious it is specific to only one type of error?

Or I guess alternatively you could attempt to make it more generic by, say, adding an `expectedErrorResult` as in `bulkCreateError` and conditionally exluding the header if the error is a `genericNotFoundEsUnavailableError`

Same nit applies to `bulkGetMgetError` and `bulkUpdateMgetError`

Reply:
> might be worth renaming this function for clarity so that it is obvious it is specific to only one type of error

I went with the renaming option